### PR TITLE
Nicoll/sks 110 implement for teams page

### PR DESCRIPTION
**Pull request by nicollguarnizo**
Total TypeScript →  https://total-typescript-turbo-git-nicoll-sks-11-63c622-skillrecordings.vercel.app/for-teams
Epic React → https://epic-react-v1-git-nicoll-sks-110-impleme-024fb1-skillrecordings.vercel.app/for-teams
Epic Web → https://epic-web-git-nicoll-sks-110-implement-fo-34b4b6-skillrecordings.vercel.app/for-teams
Testing JavaScript → https://testing-javascript-git-nicoll-sks-110-im-f8c75c-skillrecordings.vercel.app/for-teams

teams

### CODE REVIEW

File: packages/skill-api/src/core/services/send-feedback-from-user.ts [hunk 1]
```diff
@@ -42,17 +43,43 @@ export async function sendFeedbackFromUser({
       })
     }
 
-    const html = `${sanitizeHtml(feedbackText)} <i>${
-      context?.url ? context.url : ''
-    }</i>`
+    let from: string
```
Comment (nicollguarnizo) [inline packages/skill-api/src/core/services/send-feedback-from-user.ts:46]: add conditionals to send the correct data for the email 

Screenshot 2024-09-10 at 4 32 35 PM

File: packages/skill-api/src/core/types.ts [hunk 2]
```diff
@@ -50,18 +50,20 @@ export type SendFeedbackFromUserOptions = {
   feedbackText: string
   context?: FeedbackContext
   config: SkillRecordingsOptions
+  numberOfSeats?: number
 }
 
 export type FeedbackContext = {
   url?: string
-  category?: 'general' | 'help' | 'code'
+  category?: 'general' | 'help' | 'code' | 'quote requested'
```
Comment (nicollguarnizo) [inline packages/skill-api/src/core/types.ts:58]: new category for quotes

File: packages/feedback-widget/src/fields.tsx [hunk 3]
```diff
@@ -126,3 +126,94 @@ export const CategoryField: React.FC<React.PropsWithChildren<any>> = (
     </div>
   )
 }
+
+export const OptionalTextField: React.FC<React.PropsWithChildren<any>> = ({
```
Comment (nicollguarnizo) [inline packages/feedback-widget/src/fields.tsx:130]: created two new fields for the form

File: apps/epic-web/src/styles/feedback-widget.css [hunk 4]
```diff
@@ -85,3 +85,21 @@
 [data-sr-feedback-widget-category-field-buttons] button span {
   @apply after:pr-1 after:text-gray-600 after:content-['✓'] dark:after:text-gray-400;
 }
+
+/* seat selection field */
+
```
Comment (nicollguarnizo) [inline apps/epic-web/src/styles/feedback-widget.css:90]: components are going to be sharable so we can style them for each partner product

File: apps/epic-web/src/components/contact/bulk-form.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  } = useFeedbackWidget({
+    location: 'bulk-form',
+  })
+  const {data: user} = useSession()
+  const initialValues: FeedbackFormValues = {
+    email: user?.user?.email || '',
+    ...initialFeedbackFormValues,
+    context: {
+      ...initialFeedbackFormValues.context,
+      emotion: ':moneybag:',
```
Comment (nicollguarnizo) [inline apps/epic-web/src/components/contact/bulk-form.tsx:42]: directly set the context value for the bulk-form

File: packages/skill-lesson/path-to-purchase/pricing.tsx [hunk 6]
```diff
@@ -161,6 +148,38 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
...
     setQuantity,
   } = usePriceCheck()
+
+  const defaultPurchaseButtonRenderer = (
+    formattedPrice: any,
+    product: SanityProduct,
+    status: QueryStatus,
+    quantity: number,
+  ) => {
+    const isContactUs = quantity >= 30
```
Comment (nicollguarnizo) [inline packages/skill-lesson/path-to-purchase/pricing.tsx:158]: max 30 to contact us

File: packages/feedback-widget/src/fields.tsx (hunk 3 above)
Comment (vojtaholik) [reply packages/feedback-widget/src/fields.tsx:130]: I don't think we're using this package much, apps have usually their own isolated feedback widget (see TT).

**Review by joelhooks — APPROVED** [review]
looks good. we've been approaching shared component styling using a radix-style approach which would be nice here too

**Pull request merged**